### PR TITLE
Fix missing view/pure modifiers

### PR DESCRIPTION
**Pull request by sohkai**
Fixes some compiler errors.

### CODE REVIEW

File: apps/finance/contracts/Finance.sol [hunk 1]
```diff
@@ -426,7 +426,7 @@ contract Finance is AragonApp, ERC677Receiver {
         return settings.periodDuration;
     }
 
-    function getBudget(address _token) transitionsPeriod public view returns (uint256 budget, bool hasBudget, uint256 remainingBudget) {
+    function getBudget(address _token) transitionsPeriod public returns (uint256 budget, bool hasBudget, uint256 remainingBudget) {
```
[the remark this answers is not in the document]
Comment (verdverm) [reply apps/finance/contracts/Finance.sol]: What call path leads to a modified state? I'm not seeing one myself

File: apps/token-manager/contracts/TokenManager.sol [hunk 2]
```diff
@@ -205,7 +205,7 @@ contract TokenManager is ITokenController, AragonApp { // ,IForwarder makes cove
     * @param _amount The amount of the transfer
     * @return False if the controller does not authorize the transfer
     */
-    function onTransfer(address _from, address _to, uint _amount) public view returns (bool) {
+    function onTransfer(address _from, address _to, uint _amount) public returns (bool) {
```
[the remark this answers is not in the document]
Comment (verdverm) [reply apps/token-manager/contracts/TokenManager.sol]: same question as above

[the remark this answers is not in the document]
Comment (verdverm) [reply apps/token-manager/contracts/TokenManager.sol]: I found it for this one, see comment below

**Comment by verdverm** [comment]
```
    function _logHolderIfNeeded(address _newHolder) internal {
        // costs 3 sstores (2 full (20k gas) and 1 increase (5k gas)), but makes frontend easier
```

I'm not sure how I feel about charging the 25k gas to make frontend dev easier.

File: apps/finance/contracts/Finance.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (izqui) [reply apps/finance/contracts/Finance.sol]: @verdverm `getBudget` will transition to the new accounting period and return the budget in the context of this period.

@sohkai I agree that this is weird, and we should think of alternatives.

**Comment by izqui** [comment]
@verdverm I don't like it either. We plan on improving frontend loading enough so this doesn't have to be enabled when we deploy to mainnet. For beta testing now in the testnet we considered the tradeoff and we don't think it is too bad. 

We have huge plans for UI loading and state reduction improvements, but they require some work.

**Pull request merged**